### ISSUE TITLE
revisit use of prepInputs and caching

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/climateData#9]
We currently split the steps used by `prepInputs` by calling `preProcess` and `postProcess` directly, because ` doesn’t know how to work with pieces of datasets (tiles etc.) — it needs to get, assemble, then do something with the data pieces. Also, working directly with the gdal tools is faster, leaving `postProcess` to deal with final crop/mask/reproject.

We should revisit the workflow to see how we might improve it to better use `prepInputs` and `Cache` machinery, e.g. for file checking and caching.

**Comment by eliotmcintire** [comment]
This sounds like if we just put the functions in a function and pass them to `fun`, it would achieve the desired goal, no?

**Comment by achubaty** [comment]
no, because when calling prepInputs on each tile, it has no way to put them all together (e.g., mosaic) at the end

**Comment by achubaty** [comment]
@eliotmcintire if I recall correctly, you had implemented a tiled wersion of prepInputs. Did this make its way into this package?
